src/components/ActiveNote/ActiveNote.tsx

```diff
@@ -0,0 +1,19 @@
+import { Typography } from '@mui/material';
+
+import { DisplayedNote } from 'components';
+
+import { INote } from 'types';
```

Comment:
imports should be separated into 3 groups (libs, absolute and relative paths) with empty strings, 
components and types are from the second group
```suggestion
import { DisplayedNote } from 'components';
import { INote } from 'types';
```